pages/k8s/operations.md

```diff
@@ -122,6 +122,27 @@ juju config kubernetes-worker ingress=true
 Ingress resources are DNS mappings to your containers, routed through
 [endpoints](https://kubernetes.io/docs/user-guide/services/).
 
+## Configuring CoreDNS
+
+CDK 1.14+ has CoreDNS enabled by default, which allows pods within Kubernetes
+to communicate with other pods or services by name.
+
```

Comment:
Suggest to add a paragraph:

Users of CDK 1.13 and prior who are still on kube-dns can switch to CoreDNS with:

`juju config kubernetes-master dns-provider=core-dns`

Reply:
I think this might be confusing for this part of the docs. Also, it isn't quite true - users of CDK 1.13 or prior first need to upgrade. Users who WERE on 1.13 and have upgraded will hopefully have read the upgrade instructions and the notes.